client/src/components/Header/index.tsx

```diff
@@ -1,29 +1,74 @@
-import React from 'react';
+import React, { Dispatch, SetStateAction } from 'react';
 import styled from '@emotion/styled';
 
 import HeaderTop from './HeaderTop';
 import HeaderLogo from './HeaderLogo';
 import HeaderSearch from './HeaderSearch';
 import HeaderNav from './HeaderNav';
+import HeaderCategory from './HeaderCategory';
 import ICategory from '@/interfaces/category.interface';
 
 interface IHeaderProps {
   categories?: ICategory[];
+  isHover: boolean;
+  setHover: Dispatch<SetStateAction<boolean>>;
+  hoverCategory: number;
+  handleCategoryOver: ({ target }: React.MouseEvent) => void;
+  handleMouseEnter: () => void;
+  handleMouseLeave: () => void;
 }
 
-const HeaderMainWrapper = styled.div``;
+const SHeader = styled.header`
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const BottomLineWrapper = styled.div`
+  position: relative;
+  display: flex;
+  justify-content: center;
+  left: 0;
+  width: 100vw;
+  border-bottom: 1px solid lightgray;
+`;
+
+const ShadowLineWrapper = styled(BottomLineWrapper)`
+  box-shadow: 0 0.5rem 0.5rem -0.5rem rgba(0, 0, 0, 0.22);
+`;
+
+const HeaderMainWrapper = styled.div`
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  width: 100%;
+  max-width: 60rem;
+`;
 
 const Header = (props: IHeaderProps): JSX.Element => {
-  const { categories } = props;
+  const { categories, isHover, handleMouseEnter, handleMouseLeave, hoverCategory, handleCategoryOver } = props;
+
   return (
-    <header>
-      <HeaderTop />
-      <HeaderMainWrapper>
-        <HeaderLogo />
-        <HeaderSearch />
-      </HeaderMainWrapper>
-      <HeaderNav categories={categories} />
-    </header>
+    <SHeader>
+      <BottomLineWrapper>
+        <HeaderTop />
+      </BottomLineWrapper>
+      <ShadowLineWrapper>
+        <HeaderMainWrapper>
+          <HeaderNav isHover={isHover} handleMouseEnter={handleMouseEnter} handleMouseLeave={handleMouseLeave} />
```

Comment:
```suggestion
          <HeaderNav isHover={isHover} onMouseEnter={handleMouseEnter} onMouseLeave={handleMouseLeave} />
```
기본 react 컴포넌트와 비슷하게, 이벤트에는 `on~` 의 네이밍은 어떠신가요? ☺️

Reply:
저번에 페어할 때 handler를 handle~ 형식으로 적었어서 이렇게 네이밍을 한 거였습니다!
그런데 저도 on~ 네이밍 형식이 더 좋아요 😂
일단 코드는 수정하였씁니다! 혹시 모르니 다른 분들에게도 알리는 게 좋겠군용